pkg/cmdlets/commit/commit.go

```diff
@@ -0,0 +1,67 @@
+package commit
```

Comment:
I'm not confident in the naming of this package or where it lives. Let me know if this shared printing logic should be moved